tailwind.config.js

```diff
@@ -15,6 +16,7 @@ module.exports = {
       threeQuarter: '15px',
       base: '20px',
       double: '40px',
+      triple: '60px',
```

Comment:
I'm not sure if im into these names the more that we add to them 😅not something for you to change, but something we should chat about in a FE catchup or something!

Reply:
Agree! I wonder if these are valid, or if we should go for the tailwind method of basing it on `rem`.

Reply:
rem spacing added!